### PR TITLE
Auto publish to npm

### PR DESCRIPTION
**Pull request by jameshollyergoogle**
Add github action to auto publish to npm and update the version.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -28,3 +28,6 @@ jobs:
     - run: npm install
     - run: npm run build
     - run: npm run lint
+    - uses: JS-DevTools/npm-publish@v1
```
Comment (reillyeon) [inline .github/workflows/node.js.yml:31]: We want to do this in a separate workflow file which only runs on push.

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (jameshollyergoogle) [reply .github/workflows/node.js.yml]: I think I did this correctly now. I am not completely sure how to test it.

File: .github/workflows/npm_publish.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```
Comment (reillyeon) [inline .github/workflows/npm_publish.yml:16]: We probably need a "setup Node" step here:

```suggestion
    - uses: actions/checkout@v2
    - uses: actions/setup-node@v1
        with:
          node-version: 10
```

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (reillyeon) [reply .github/workflows/node.js.yml]: I don't think there's a good way to test. We just need to land it, bump the version and see what happens. Have you set up the NPM token secret?

**Pull request merged**